Trappist/src/Promact.Trappist.Web/wwwroot/app/questions/questions-programming/questions-programming.html

```diff
@@ -73,14 +73,14 @@
                 <div class="form-group">
                     <label class="h4">Test Case(s)</label>
                     <br />
-                    <button type="button" class="btn btn-primary" [hidden]="formControlModel.showTestCase" (click)="formControlModel.showTestCase=true">Add Test Case</button>
-                    <md2-accordion>
+                    <button type="button" class="btn btn-primary" [hidden]="formControlModel.showTestCase" (click)="formControlModel.showTestCase=true;addTestCases()">Add Test Case</button>
+                    <md2-accordion *ngFor="let testCase of testCases;let i = index">
```

Comment:
here I refers to the index, its significance is to store the number of iterations that it is doing, if its not implemented then while adding new test case i.e. pushing a new object, the data of the previous test case disappears, to fix that issue it is used.